README.md

```diff
@@ -20,30 +20,30 @@
 
 # Prefect
 
-Prefect is an orchestration and observability platform for building, observing, and triaging workflows.
+Prefect is an orchestration and observation platform for building and reacting to workflows.
```

Comment:
Interesting, I don't think "observation" is wrong but it is unconventional. Unless you feel strongly about it, I'd srick with "observability"